### PR TITLE
Enable more sqllogictest files for -bigtest.

### PR DESCRIPTION
**Pull request by petermattis**

### CODE REVIEW

**Comment by vivekmenezes** [comment]
LGTM

File: sql/logic_test.go [hunk 1]
```diff
@@ -478,7 +500,11 @@ func TestLogic(t *testing.T) {
 		paths = append(paths, match...)
 	}
 
+	total := 0
 	for _, p := range paths {
-		logicTest{T: t}.run(p)
+		l := logicTest{T: t}
+		l.run(p)
```
Comment (tamird) [inline sql/logic_test.go]: nit: have `run` return the progress?

**Comment by tamird** [comment]
LGTM

**Comment by MBerhault** [comment]
LGTM. arg! the sqllogictest repo is huge.

**Pull request merged**